locals.tf

```diff
@@ -1,7 +1,8 @@
 locals {
+  default_name     = "${var.env}-${var.namespace}"
```

Comment:
I would not have two "name" variables. How default name is different from name? (also do we really need `-tcp` suffix in name? 

I'd say we should try to optimize it so we have only one name. Or just remove the default_name local?